commit cc2f3cdecff5a351e7e8961b9f2e389ab740231a
Author: Ewald Hew <ewaldhew@gmail.com>
Date:   Wed Jan 10 13:24:56 2018 +0800

    [psaux] Correctly handle Flex features (#52846).
    
    * src/psaux/psintrp.c (cf2_interpT2CharString) <cf2_cmdVMOVETO,
    cf2_cmdHMOVETO>: Do not move if doing Flex.

diff --git a/src/psaux/psintrp.c b/src/psaux/psintrp.c
index 5c0ee7849..da5a8dad1 100644
--- src/psaux/psintrp.c
+++ src/psaux/psintrp.c
@@ -852,7 +852,8 @@
 
         curY = ADD_INT32( curY, cf2_stack_popFixed( opStack ) );
 
-        cf2_glyphpath_moveTo( &glyphPath, curX, curY );
+        if ( !decoder->flex_state )
+          cf2_glyphpath_moveTo( &glyphPath, curX, curY );
 
         break;
 
@@ -2674,7 +2675,8 @@
 
         curX = ADD_INT32( curX, cf2_stack_popFixed( opStack ) );
 
-        cf2_glyphpath_moveTo( &glyphPath, curX, curY );
+        if ( !decoder->flex_state )
+          cf2_glyphpath_moveTo( &glyphPath, curX, curY );
 
         break;
 
